ivy/functional/backends/tensorflow/statistical.py

```diff
@@ -12,17 +12,31 @@
 # -------------------#
 
 
-@with_unsupported_dtypes({"2.14.0 and below": ("complex",)}, backend_version)
+@with_unsupported_dtypes(
+    {"2.14.0 and below": ("complex", "bool", "uint64")}, backend_version
```

Comment:
not sure why `tf.mathreduce_min` doesn't support `uint64`, their docstring doesn't seem to indicate so, but it seems to be something that should ideally work, have you tested it with an example?